docs/utils/SolidityLexer.py

```diff
@@ -68,16 +67,11 @@ class SolidityLexer(RegexLexer):
                  ['fixed%dx%d' % ((i), (j + 8)) for i in range(0, 256, 8) for j in range(0, 256 - i, 8)]
```

Comment:
In `fixedMxN` `N` refers to the number of decimal digits now as opposed to bit width. However as I see it is not yet updated anywhere so probably fine to keep.